### ISSUE TITLE
`constellation config generate` creates broken configuration file

### ISSUE THREAD

**Issue by malt3** [edgelesssys/constellation#1653]
### Issue description

When creating a Constellation on host systems that are not Linux or not amd64, `constellation config generate` would generate a broken microservice version:

```
Problems validating config file:
    image specifies an invalid version: configured version (v2.7.0) does not adhere to SemVer syntax
    microserviceVersion specifies an invalid version: configured version (v) does not adhere to SemVer syntax
Fix the invalid entries or generate a new configuration using `constellation config generate`
Error: invalid configuration
```

`constellation-conf.yaml`:
```
version: v2 # Schema version of this configuration file.
image: v2.7.0 # Machine image version used to create Constellation nodes.
[...]
microserviceVersion: v # Microservice version to be installed into the cluster. Defaults to the version of the CLI.
[...]
```

### To reproduce

1. Download the Constellation CLI in version 2.7.0 for a platform that is not linux/amd64
    ```
    wget https://github.com/edgelesssys/constellation/releases/download/v2.7.0/constellation-darwin-arm64
    chmod +x constellation-darwin-arm64
    ```
2. Generate a config file for any cloud provider: `constellation config generate gcp`
3. Look a the `microserviceVersion` field in the generated config file.
4. You can spot the issue by running `constellation version`, where the version field only shows the kind of build (`Enterprise`) but no actual version information
    ```
    Version:	 (Enterprise build; see documentation for license agreement)
    GitCommit:	d56b0ef75f5a5019f608f4907d1cbcea4b749799
    GitTreeState:	clean
    BuildDate:	1970-01-01T00:00:00Z
    GoVersion:	go1.20.2 X:nocoverageredesign
    Compiler:	bazel/gc
    Platform:	linux/arm64
    ```

### Environment

- `constellation version`: v2.7.0
- Host platform that is not linux/amd64

### Expected behavior

The config file should be generated correctly and it should be possible to create a Constellation with it.

### Mitigation

Use the Constellation cli on linux/amd64.

**Comment by katexochen** [comment]
This was fixed in #1632 and will be part of the upcoming v2.7.1 patch release.

**Comment by malt3** [comment]
Fixed in [v2.7.1](https://github.com/edgelesssys/constellation/releases/tag/v2.7.1)
